OPS-59: Add NZZO and Oceanics (#66)
vatnz-dev/sops: docs/oceanic/index.md

| YMMM - Melbourne Oceanic             | Melbourne Radio     | 17.904 (124.650) | ML-IND_FSS   | VATPAC   |                             |
| NZZO - Auckland Oceanic              | Auckland Radio      | 8.867 (129.000)  | NZZO_FSS     | VATNZ    | Can be split into -E sector |
| NZZO-TSN - Auckland Oceanic (Tasman) | Auckland Radio      | 5.567 (129.500)  | NZZO-TSN_FSS | VATNZ    |                             |
| NZCM - McMurdo Oceanic               | McMurdo Radio       | 9.032 (128.700)  | NZCM_FSS     | VATNZ    |                             |
| NFFF - Nadi Oceanic                  | Nadi Radio          | 8.846 (123.600)  | NFFF_FSS     | VATPAC   |                             |
| NTTT - Tahiti Oceanic                | Tahiti Radio        | 9.116 (125.500)  | NTTT_FSS     | VATPAC   |                             |
| RJTG - Tokyo Oceanic                 | Tokyo Radio         |                  |              | VATJPN   |                             |
| MMFO - Mexico                        |                     |                  |              |          |                             |

### Currency Requirements

> In order to utilize your Oceanic Endorsement, and by extension, exersize your rights under the Pacific Oceanic Agreement, you must meet the following criteria:

Reviewer: ```suggestion
In order to utilise your Oceanic Endorsement, and by extension, exercise your rights under the Pacific Oceanic Agreement, you must meet the following criteria:
```

> In order to utilise your Oceanic Endorsement, and by extension, exersize your rights under the Pacific Oceanic Agreement, you must meet the following criteria:

Reviewer: ```suggestion
In order to utilise your Oceanic Endorsement, and by extension exercise your rights under the Pacific Oceanic Agreement, you must meet the following criteria:
```

> - A Participating Organisation must require a permitted controller to log 1 hour on a permitted oceanic position, within the previous 3 months, for them to be considered active, and

Reviewer: ```suggestion
- A Participating Organisation must require a permitted controller to log 1 hour on a permitted oceanic position, within the previous 3 months, for them to be considered active; and
```

- A permitted controller who does not meet the hour requirement specified above, will be considered inactive; and

> - An inactive permitted controller may become active again, by performing a buddy session with a permitted controller who is approved to do so by that PO; and

Reviewer: I'm not sure PO is previously mentioned anyway so might be a bit confusing and require expanding on first time round?

Reviewer 2: could change to "participating division" as mentioned in ## Pacific Oceanic Agreement?

| RJTG - Tokyo Oceanic                 | Tokyo Radio         |                  |              | VATJPN   |                             |
| MMFO - Mexico                        |                     |                  |              |          |                             |

!!! warning "Use of the /O Suffix"
    Controllers are no longer required to use the `/O` suffix when logging onto Oceanic positions. This is due to the Division Controller Roster fulfilling the requirements for publishing a Controller's right to control Oceanic positions.

### Currency Requirements

> In order to utilise your Oceanic Endorsement, and by extension, exersise your rights under the Pacific Oceanic Agreement, you must meet the following criteria:

Reviewer: ```suggestion
In order to utilise your Oceanic Endorsement, and by extension, exercise your rights under the Pacific Oceanic Agreement, you must meet the following criteria:
```
